### ISSUE TITLE
Meteor.JS install

### ISSUE THREAD

**Issue by mroftalp** [Nickersoft/push.js#63]
How to install this on meteor.js? I tried the npm install and insert this inside my app

`import moment from 'push.js';
const Push = require('push.js');`

But the Push is still undefined..

**Comment by Nickersoft** [comment]
Hmm... I'm actually not too familiar with Meteor, though it theoretically should work. Perhaps someone else might be able to offer assistance. Adding a "help wanted" tag.

**Comment by mroftalp** [comment]
Yeah there's something that I have not done correctly. This is actually my first time trying npm_modules in meteor.

**Comment by rafinskipg** [comment]
@mroftalp import * as Push from 'push.js';

**Comment by Molosc** [comment]
Hi
I can import the package into Meteor with npm but I get a "Push.create is not a function" error when I fire it.

Push exist, but with just this properties inside:

```

Permission:e {_win: Window, DEFAULT: "default", GRANTED: "granted", DENIED: "denied", _permissions: Array(3)}
default:e {_currentId: 0, _notifications: {…}, _win: Window, Permission: e, _agents: {…}, …}
Symbol(__esModule):true
_agents:{desktop: t, chrome: t, firefox: t, ms: t, webkit: t}
_configuration:{serviceWorker: "/serviceWorker.min.js", fallback: ƒ}
_currentId:0
_notifications:{}
_win:Window {stop: ƒ, open: ƒ, alert: ƒ, confirm: ƒ, prompt: ƒ, …}
Symbol(Symbol.toStringTag):"Module"

```

Any idea? Should I do something more to import correctly @rafinskipg ?
Do you made it work with Meteor @mroftalp ?
It seem to work great! I need it :D 

Thank you for help

**Comment by mroftalp** [comment]
@Molosc First, install push. Type this command on your project directory through the terminal. `meteor npm install push.js --save`. Then when importing, it should be like this: `import Push from 'push.js';`. That's how I made it work with Meteor.
